cookbooks/aws-parallelcluster-config/recipes/aws_cli.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster
```

Comment:
wrong cookbook name

Reply:
Not part of this PR but I'll fix. thank you